common/logging/log4rs-sample.yml

```diff
@@ -46,7 +83,31 @@ loggers:
 
   # Route log events sent to the "comms" logger to the "network" appender
   comms:
-    level: info
+    level: trace
+    appenders:
+      - network
+    additive: false
+  # Route log events sent to the "p2p" logger to the "network" appender
+  p2p:
+    level: trace
     appenders:
       - network
-    additive: false
\ No newline at end of file
+    additive: false
+ # Route log events sent to the "mio" logger to the "other" appender
+  mio:
```

Comment:
This should also be in comms as well. This is something to filter out with lnav because it has no meaning without the comms logs

Reply:
moved this over to the network log